### ISSUE TITLE
Wrong args

### ISSUE THREAD

**Issue by Hongarc** [pcottle/learnGitBranching#506]
I type: (2 space in `git` and `checkout`)
```bash
    git  checkout myBranch
````
but it return 2 args of checkout command: `checkout` and `myBranch` (args must be `myBranch`) so `terminal` log:

> I expected at most 2 argument(s) for with git checkout

We need remove git command (`checkout `, `commit`, etc) from args when type many space.

**Comment by pcottle** [comment]
yeah good point, this doesnt seem like a super urgent fix but ill try to get to it eventually
